### PR TITLE
[10.0] Migrate mrp_production_partner_note

### PR DESCRIPTION
**Pull request by bimajatiwijaya**
1. edit parameter name and adjust code : _prepare_mo_vals(self, [now it from mrp.bom object])
2. change product on unit test use existing product ..5b

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]
LGTM.
Good as is but would be appreciated if you could raise the coverage up a bit

File: mrp_production_partner_note/__manifest__.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
         "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
         "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
         "Ana Juaristi <anajuaristi@avanzosc.es>",
+        "Bima Wijaya <bimajatiwijaya@gmail.com>",
```
Comment (andhit-r) [inline mrp_production_partner_note/__manifest__.py]: Remove contributor key

**Comment by bimajatiwijaya** [comment]
Any advice to increase coverage ?
I have no experienced for this :)

**Comment by pedrobaeza** [comment]
You have to develop tests that exercises the code of this module.

**Comment by LoisRForgeFlow** [comment]
Closing as there are unanswered requested changes.

**Pull request closed without merging**